pkg/arvo/neo/cod/std/src/fil/types.txt

```diff
@@ -0,0 +1,460 @@
+# Shrubbery Types
+
+This doesn't cover every type defined in `/sur/neo.hoon`. It does cover all of the types you'll regularly work while writing userspace shrubs, and the types you need to know about to fully understand those types.
+
+## `aeon`
+
+Total version numbers and signature for a shrub. [How is the signature being used here?]
+
+```
++$  aeon  (pair ever oath)
+```
+
+## `axal`
+
+Fundamental node, recursive.
+
+This mold builder creates a representation of a node in Clay like an [arch](https://docs.urbit.org/language/hoon/reference/arvo#arch) (a file or a directory) or [axil](https://docs.urbit.org/language/hoon/reference/arvo#axil) (fundamental node), but the directory map contains more axals, so it contains the entire subtree rather than just one level.
+
+```
+++  axal
+  |$  [item]
+  [fil=(unit item) kid=(map iota $)]
+```
+
+## `band`
+
+Dependency map.
+
+```
++$  band  $+(band (map term fief))
+```
+
+The `term`s are arbitrary keywords defined in a shrub's `+deps` arm. When that shrub is created, the `%make` card's `conf` will have to include a map of all the required dependencies.
+
+## `bowl`
+
+`bowl:neo`, similar to `bowl:gall`.
+
+```
++$  bowl
+  $:  src=name
+      our=@p
+      were=pith
+      here=pith
+      now=@da
+      eny=@uvJ
+      deps=(map term (pair pith lore))
+      kids=lore
+  ==
+```
+
+- `src`: source ship and path [of what?].
```

Comment:
the change that woke the shrub up. For +init, this is what shrub issued the %make, and for +poke this is the shrub that issued the %poke